src/crimson/osd/osd_operations/pg_splitting.cc

```diff
@@ -53,63 +48,64 @@ void PGSplitting::dump_detail(Formatter *f) const
 
 seastar::future<> PGSplitting::start()
 {
-  logger().debug("{}: start", *this);
+  LOG_PREFIX(PGSplitting::start);
+  DEBUGDPP("{}: start", *this);
   unsigned old_pg_num;
   using cached_map_t = OSDMapService::cached_map_t;
 
   if (cur_map->have_pg_pool(pg->get_pgid().pool())) {
     old_pg_num = cur_map->get_pg_num(pg->get_pgid().pool());
   } else {
-    logger().debug("{} pool doesn't exist in {}", pg->get_pgid(), cur_map);
+    DEBUGDPP("{} pool doesn't exist in epoch {}", pg->get_pgid(), cur_map->get_epoch());
```

Comment:
DEBUGDPP expects a context prefix as well:
```
DEBUGDPP("{} pool doesn't exist in epoch {}", *pg, pg->get_pgid(), cur_map->get_epoch());
                                              ^^^
```

When context prefix is not needed `DEBUG` can be used instead

Reply:
Ah thank you, I'll make the change